{{cookiecutter.project_slug}}/home/management/commands/createsuperuserauto.py

```diff
@@ -11,18 +12,30 @@ def add_arguments(self, parser):
             '--password', dest='password', default=None,
             help='Specifies the password for the superuser.',
         )
+        if self.UserModel.USERNAME_FIELD != "username":
+            parser.add_argument(
+                '--username', dest='username', default=None,
+                help="Specifies the username for the superuser"
+            )
 
     def handle(self, *args, **options):
         password = options.get('password')
         username = options.get('username')
         database = options.get('database')
+        email = options.get('email')
 
         if password and not username:
             raise CommandError("--username is required if specifying --password")
```

Comment:
Looks like this should be removed since it's checked at line 33

Reply:
Yes, forgot to remove this.